Parkinsons-Mouse-tutorial.ipynb

```diff
@@ -0,0 +1,3062 @@
+    "\n",
+    "We’ll look a subset of data from two human donors (one healthy and one with PD) whose samples were each transplanted into three separate cages of mice from the susceptible genotype. For this tutorial, a subset of the metadata has been prepared, and the sequences have been subsampled to approximately 5000 sequences per sample to allow the tutorial to run in a short time. The sequences for the full study are accessible at EBI with accession [PRJEB17694](https://www.ebi.ac.uk/ena/data/view/PRJEB17694); processed tables from the full study can be downloaded from the [Qiita](https://qiita.ucsd.edu/) database from study 10483."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Hypothesis"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial will explore the hypothesis that the genetic background of a humanized mouse influences the microbial community. However, we’ll also need to consider other confounders which might drive the shape of the microbiome instead of the mouse genotype."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Importing necessary modules"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import qiime2"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiime2.plugins import (demux, dada2, metadata, feature_table,\n",
+    "                            fragment_insertion, diversity, \n",
+    "                            longitudinal, phylogeny, sample_classifier,\n",
+    "                            feature_classifier, taxa, composition, \n",
+    "                            longitudinal)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Set up"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This tutorial assumes that you have QIIME 2 installed according to the [installation instructions](https://docs.qiime2.org/2020.6/install/).\n",
+    "\n",
+    "Before running the tutorial, you will need to make a directory for the tutorial data and navigate into that directory."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "workdir='/home/user/Documents/qiime2-mouse-tutorial/'"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "!mkdir -p $workdir\n",
+    "!cd $workdir"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Metadata"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Before starting any analysis, it’s important to be familiar with the metadata. In this study, the metadata file contains 7 columns."
```

Comment:
Czemu przed zaimportowaniem metadanych importujesz manifest file i to bez żadnej informacji, co to jest i po co nam to potrzebne? Manifest file powinnien być w akapicie "Importing metadata as a qiime2 artifact"